sass/components/_mixins.scss

```diff
@@ -2,4 +2,17 @@
 //     -webkit-box-shadow: $args1, $args2;
 //     -moz-box-shadow: $args1, $args2;
 //     box-shadow: $args1, $args2;
-// }
\ No newline at end of file
+// }
+
+@mixin not-selectable {
+	-webkit-touch-callout: none;
```

Comment:
Are browser specific prefixes required here? MateralizeCSS uses autoprefixer to add browser-dependent CSS https://github.com/Dogfalo/materialize/blob/master/Gruntfile.js#L81

You can try to remove it, build and see what prefixes would be added automatically.